automation.py

```diff
@@ -0,0 +1,16 @@
+
+import unittest
```

Comment:
Did you consider using pytest? its far easier to use then unittest (you need less boilerplate code), and it can produce test results that are far easier to read and debug with.

Was using unittest a team decision?

Reply:
I recommend using `pytest` since this is the tool we will use in this project and as @ifireball said, much more easier. But, it's not a blocker for now and can be changed when we learn `pytest`.